aerleon/lib/ciscoxr.py

```diff
@@ -58,7 +61,7 @@ def _BuildTokens(self):
 
         return supported_tokens, supported_sub_tokens
 
-    def _GetObjectGroupTerm(self, term, verbose=True):
+    def _GetObjectGroupTerm(self, term: Term, verbose: bool = True) -> "CiscoXRObjectGroupTerm":
```

Comment:
Any idea why CiscoXRObjectGroupTerm appears in quotes? Not a problem but it doesn't seem to match the rest of the PR.

Reply:
I think this just means the type has not yet been defined in this file.

Reply:
Ahh monkeytype has been doing that in some cases and not others. I try to fix it where I see it. Thanks for catching it.